utils/markdown/autolink.go

```diff
@@ -59,9 +60,8 @@ func isAllowedBeforeWWWLink(c byte) bool {
 	switch c {
 	case '*', '_', '~', ')':
 		return true
-	default:
-		return false
 	}
+	return false
```

Comment:
Interesting. Is returning outside the switch more idiomatic?

Reply:
I would say so.

```go
func IsSpace(r rune) bool {
  // This property isn't the same as Z; special-case it.
  if uint32(r) <= MaxLatin1 {
    switch r {
    case '\t', '\n', '\v', '\f', '\r', ' ', 0x85, 0xA0:
      return true
    }
    return false
  }
  return isExcludingLatin(White_Space, r)
}
```

The core idea is to reduce indentation as much as possible.

Reply:
I was thinking the opposite :) Somehow it looks more clear to me with the default case if the function/method only does a `switch`.

Reply:
Hmm .. I can remove it if you want. But in general, the convention has always been to try to reduce amount of code written with indentation. So as soon as the need for an indentation finishes, we can come back to our original indent. For the switch, I think of it as a compressed if block.

```
if x,y,z {
 return true
}
return false
```

Reply:
Yeah it makes total sense for `if` statements. And I see this occasion looks like a compressed `if` block. It may be a personal preference though but I tend to add a `default` case if I ever use switch :) And since for this specific case this is totally cosmetic, I leave it to you.